src/main/java/org/wso2/carbon/module/swiftiso20022/validation/common/ValidationResult.java

```diff
@@ -16,49 +16,40 @@
  * under the License.
  */
 
-package org.wso2.carbon.module.swiftiso20022.model;
+package org.wso2.carbon.module.swiftiso20022.validation.common;
 
 /**
  * Class to hold the error details.
  */
-public class ErrorModel {
-    private boolean isError;
+public class ValidationResult {
+    private boolean isValid;
```

Comment:
Don't we need a setter method for this field?

Reply:
From the current perspective, we don't need the setter methods.